orm/src/main/java/org/hibernate/search/query/hibernate/impl/SecondLevelCacheObjectInitializer.java

```diff
@@ -38,7 +38,7 @@ public void initializeObjects(List<EntityInfo> entityInfos, LinkedHashMap<Entity
 		final int maxResults = entityInfos.size();
 		if ( maxResults == 0 ) {
 			if ( traceEnabled ) {
-				log.tracef( "No object to initialize", maxResults );
+				log.tracef( "No object to initialize", (Integer) maxResults );
```

Comment:
Good points :+1: 

Now we need a validation plugin to find mismatches like these ;)